DOM I Repurpose/js/index.js

```diff
@@ -0,0 +1,204 @@
+
+mainContentP[2].innerHTML = (siteContent['main-content']['product-content']);
+mainContentP[3].innerHTML = (siteContent['main-content']['vision-content']);
+
+//footer
+
+docH4[5].innerHTML = (siteContent['contact']['contact-h4']);
+
+mainContentP[5].innerHTML = (siteContent['contact']['address']);
+mainContentP[6].innerHTML = (siteContent['contact']['phone']);
+mainContentP[7].innerHTML = (siteContent['contact']['email']);
+
+//copyright
+
+mainContentP[8].innerHTML = (siteContent['footer']['copyright']);
+
+// DOM II Assignment
+
+// * Create 10 unique event listeners based on the reference file in the objective.  
+
+// 1. changes cta font to red on button click
+let grabButton = document.querySelector('button');
+
+let grabMainCTA = document.querySelector('.cta-text h1');
+
+grabButton.addEventListener('click', () => {
+  grabMainCTA.style.color = 'red';
+});
+
+
+// 2. on mousing over cta image, cta image disapears cta copy is reduced to 10px 
+let grabCTAimage = document.querySelector('#cta-img');
+
+grabCTAimage.addEventListener('mouseenter', () => {
+  grabCTAimage.style.display = 'none';
+  grabMainCTA.style.fontSize = '10px';
+})
+
+// 3. cta copy is set to 200px on mousing over and out cta copy 
+grabMainCTA.addEventListener('mouseleave', () => {
+  grabMainCTA.style.fontSize = '200px';
+})
+
+// 4. cta copy is set to 10px when window object is resized 
+window.addEventListener('resize', () => {
+  grabMainCTA.style.fontSize = '9px';
+})
+
+ 
+// 5. whenever a keyboard key is pressed an alert appears showing what key was pressed
+
+document.addEventListener('keydown', (event) => {
+  const keyName = event.key;
+  alert('key: ' + keyName);
+});
+
+
+// 6. if you place the mouse over the middle image and try to right click a message will be displayed in an alert.
+
+let grabMiddelImg = document.querySelector('#middle-img');
+
+grabMiddelImg.addEventListener('contextmenu', () => {
+  alert(`Don't even think about lifting this image!`);
+})
+
+
+// 7.  if you copy anything from page and then try to paste it, the copied content will not appear, rather , a special message will appear instead.
+
+document.addEventListener('copy', function(e){
+  e.clipboardData.setData('text/plain', 'Hello, Mr. Chris Honda');
+  e.clipboardData.setData('text/html', '<b>Hello, Mr. Chris Honda!</b>');
+  e.preventDefault(); // We want our data, not data from any selection, to be written to the clipboard
+});
+
+
+// 8. On initial page load  an alert will appear
+
+window.addEventListener('load', () => {
+  alert('Prepare to be amazed by another Lambda School project!')
+})
+
+// 9. on window scroll background color of container will change
+
+let pageContainer = document.querySelector('.container')
+
+window.addEventListener('scroll', () => {
+  pageContainer.style.backgroundColor = "violet"
+});
+
+
+// 10. Upon double clicking anywhere in the document the background will turn white
+
+document.addEventListener('dblclick', () => {
```

Comment:
I like that you gave me a way to effectively undo another event listener's changes.